ffi/structs/c/Makefile

```diff
@@ -7,5 +7,8 @@ structs.o: structs.c
 dylib: structs.o
 	gcc -dynamiclib -undefined suppress -flat_namespace structs.o -o ../structs.dylib
 
+dll: structs.o
+	gcc -shared structs.o -o ../structs.dylib
```

Comment:
shouldn't this be dll?

Reply:
I would just use `Platform.isWindows`/`Platform.isMac` to determine what the suffix should be before calling `DynamicLibrary.open`.

But I think it's OK to leave .dylib as well, since it's just a syntactic detail.

Reply:
I've changed it to use `Platform.isWindows` for now.